capstone/src/main/webapp/dashboard.html

```diff
@@ -13,9 +13,59 @@
     <!--Let browser know website is optimized for mobile-->
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
   </head>
-  <body>
-     <h3>Dahsboard<h3>
+  <body onload="getCurrentUserInfo()">
+     <nav class="nav-wrapper">
+        <a href="" class="brand-logo">Studentia</a>
+        <a href="#" class="sidenav-trigger" data-target="slide_out" ><i class="material-icons">menu</i></a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+          <li><a href="findStudySet.html">Search</a></li>
+          <li><a href="/">Home</a></li>
+        </ul>
+    </nav>
+
+    <ul class="sidenav" id="slide_out">
+          <li><a href="findStudySet.html">Search</a></li>
+          <li><a href="index.html">Home</a></li>
+    </ul>
+
+
+    
+    <div class="container">
+        <blockquote class="flow-text">Welcome to your personal dashboard, start learning !! </blockquote>
+        <div class="row">
+            <div class="col s12 m6 l6 responsive-spaced-box z-depth-5">
+                    <h4>Your Profile Details</h4>
+                    <hr>
+                    <p>Full Name: <span id="full_name"></span></p>
+                    <p>User Name: <span id="user_name"></span></p>
+                    <p>University: <span id="university"></span></p>
+                    <p>Email Address: <span id="email"></span></p>
+                    <a href="" id="logout_url"  class="btn waves-effect waves-light">Log Out</a>
+                    <br>
+            </div>
+            <div class="col s12 m12 l6 responsive-spaced-box z-depth-5">
+                    <p class="flow-text">
+                        Start studying and helping others by creating your study set now ! <br>
+                    </p>
+                    <a href="" id="logout_url"  class="btn waves-effect waves-light">
+                        <i class="material-icons right">add_circle</i> Create study set
+                    </a>
+                    <br>
+            </div>
+        </div>
+    </div>
+
+
+    <footer class="page-footer">
+       <div class"container">
+           © 2020 Studentia inc
+       </div>
+    </footer>
+
      <!--JavaScript at end of body for optimized loading-->
-     <script type="text/javascript" src="js/materialize.min.js"></script>
+    <script type = "text/javascript" src = "https://code.jquery.com/jquery-2.1.1.min.js"></script>         
+    <script type="text/javascript" src = "https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/js/materialize.min.js"></script> 
+    <script src="script.js"></script>  
+    <script src="script2.js"></script>
```

Comment:
I like the idea of breaking up the script files, but give them more descriptive file names so you know what logic lives in which file